themes/src/main/resources/theme/base/login/login.ftl

```diff
@@ -1,9 +1,17 @@
 <#import "template.ftl" as layout>
 <@layout.registrationLayout displayInfo=social.displayInfo; section>
     <#if section = "title">
-        ${msg("loginTitle",(realm.displayName!''))}
```

Comment:
I would ask you to create a separated JIRA for this one. The reason is that those changes are beyond the scope of KEYCLOAK-3446 (pt_BR localization).